addons/test_mail/security/ir.model.access.csv

```diff
@@ -67,3 +67,4 @@ access_mail_test_properties_user,mail.test.properties.user,model_mail_test_prope
 access_mail_test_mail_tracking_duration,mail.test.mail.tracking.duration,model_mail_test_mail_tracking_duration,base.group_user,1,1,1,1
 access_mail_test_public_user,mail.test.public.user,model_mail_test_public,base.group_user,1,1,1,1
 access_mail_test_public_public,mail.test.public.public,model_mail_test_public,base.group_public,1,1,0,0
+access_mail_test_alias_domain,mail.test.alias.domain,model_mail_test_alias_domain,base.group_user,1,1,1,1
```

Comment:
To define before access_mail_test_alias_optional_portal (we try to keep alphabetical ordering, even if not complete currently)